template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/EmptyWeb-CSharp/Company.WebApplication1.csproj

```diff
@@ -7,6 +7,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Update="Program.Designer.cs">
+      <DependentUpon>Program.cs</DependentUpon>
```

Comment:
Yes, but it'll taint the project--the item will become a sibling of Program.cs instead of nested.